domains/sudip.json

```diff
@@ -7,6 +7,6 @@
         "twitter": "sudipkundu999"
     },
     "record": {
-        "URL": "https://sudipkundu999.github.io/sudipkundu"
+        "CNAME": "sudipkundu999.github.io"
```

Comment:
Got to have a bit more content than that.

Reply:
Hey, this is the site, I want to host on sudip.is-a.dev - 
https://sudipkundu999.github.io/sudipkundu/

issue was that both were redirecting to each other

Reply:
@DEV-DIBSTER can you please help here?

Reply:
Adding more context,

GitHub pages site for any given repo is
`https://<user-name>.github.io/<repo-name>`
Each of these pages are separate GitHub pages to Github even though it's under same domain.

so a user can have as many GitHub pages as they want but the domain will be the same and the path will be different.

so CNAME for all individual GitHub pages of a user will be the same - `<user-name>.github.io`